dao/mongo.go

```diff
@@ -167,3 +168,24 @@ func (m *MongoService) CheckMultipleCorporateBodySubmissions(companyNumber strin
 
 	return true, nil
 }
+
+// CheckMultipleUserSubmissions checks whether for multiple user submitted requests.
```

Comment:
Wording on this is a bit strange, would change to something like 
`// CheckMultipleUserSubmissions checks whether a user has submitted multiple requests.`